feat: setup README and add LICENSE

DLineone opened the pull request:
setup README

- Header
- Table of contents
- About the project
- Build with
- Getting started
  - Prerequisits
  - Instalation
  - Development
  - Deploy
- Footer

also added license

DLineone committed:
feat(docs): setup README

- Header
- Table of contents
- About the project
- Build with
- Getting started
-- Prerequisits
-- Instalation
-- Development
-- Deploy
- Footer

DLineone committed:
feat(docs): add license

alarmichanin commented on LICENSE.txt:
The Unlicense
Why did you decide to use this one?
I think, it's better to use MIT License with the future commercialisation (read about it MIT License + Open Core model)
After we will change it to commercial one

alarmichanin commented on README.md:
```diff
@@ -1 +1,255 @@
+<!-- Improved compatibility of back to top link: See: https://github.com/othneildrew/Best-README-Template/pull/73 -->
+<a id="readme-top"></a>
+<!--
+*** Thanks for checking out the Best-README-Template. If you have a suggestion
```
Do we need all this comment staff? 
Or we should use it cause of license?

alarmichanin commented on LICENSE.txt:
```diff
@@ -0,0 +1,9 @@
+The Unlicense
```
As a company you can use CYF naming (create your future)
Logo and etc I will add later

alarmichanin commented on README.md:
````diff
@@ -1 +1,255 @@
+   source .venv\Scripts\activate
+   pip install -r requirements.txt
+   ```
+4. Create .env file (example)
````
You should left line **Create .env file**
We will left .env file in repo without value as an example inside the project 
And for development we will use .env.local (I will show you after)

alarmichanin commented on README.md:
````diff
@@ -1 +1,255 @@
+
+1. Run client
+   ```sh
+   npm run dev
````
I think firstly we will create TG bot, so no needs in FE for the start

alarmichanin commented on README.md:
````diff
@@ -1 +1,255 @@
+   ```
+2. Start docker container
+   ```sh
+   docker-compose up
````
Can we use development through docker? Without local configuration and setup?

alarmichanin commented on images/logo.png:
Good one! I like it!

alarmichanin reviewed (changes requested):
GJ bro, add some changes

DLineone committed:
fix(docks): fix readme and license pr issues

- change license
- remove unnecessary comments
- change build and deploy section
